examples/hello-world/helloworld.yaml

```diff
@@ -53,17 +74,19 @@ topology_template:
       properties:
         resource_id: aria_helloworld_floatingip
         create_if_missing: true
+        openstack_config: { get_input: openstack_config }
         floatingip:
-          floating_network_id: 6751cb30-0aef-4d7e-94c3-ee2a09e705eb # should change to: { get_input: external_network_id }
+          floating_network_id: { get_input: floating_network_id }
```

Comment:
floating_network_name as a top level property should work, then you can get-input external network